packaging/nnstreamer.spec

```diff
@@ -261,10 +261,11 @@ popd
 %post -n nnstreamer-python2
 ln -s %{_prefix}/lib/nnstreamer/filters/nnstreamer_python2.so %{python_sitelib}/nnstreamer_python.so
 
-%postun -p /sbin/ldconfig
+%postun
 pushd %{_libdir}
 rm libnnstreamer.so
 popd
+/sbin/ldconfig
```

Comment:
Be careful. Uninstalling things with postun is more difficult than you think:
https://ma.ttias.be/on-removing-users-with-postun-in-rpm-spec-files/

For such a symlink, it's be better done at %install
E.g.,
```
%install
...
...
pushd %{buildroot}%{_libdir}
ln -sf %{gstlibdir}/libnnstreamer.so libnnstreamer.so
popd
...
...
```

This applies to other symlinks as well.

Reply:
OK. I will.